examples_tests/35.TAA/velbuf.vert

```diff
@@ -0,0 +1,22 @@
+#version 430 core
+
+layout (location = 0) in vec3 vPos;
+
+layout (location = 0) uniform mat4 MVP;
+layout (location = 1) uniform mat4 prevMVP;
+
+out vec2 vVelocity;
+
+void main()
+{
+	vec4 p_cs = MVP*vec4(vPos, 1.0);
```

Comment:
define a FAST_TRANSFORM macro for future use `Matrix[0]*vec.x+Matrix[1]*vec.x+Matrix[2]*vec.x+Matrix[3]`